qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/create/CreateCustomerOutput.groovy

```diff
@@ -11,4 +11,6 @@ import life.qbic.portal.portlet.UseCaseFailure
  */
 interface CreateCustomerOutput extends UseCaseFailure {
 
+    void successNotification(String notification)
```

Comment:
added the `successNotification` to prevent future errors due to calls of this method (was part of the `UseCaseFailure` interface before)